Tests/Microsoft.Azure.Mobile.Test.Windows/StartServiceLogTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Microsoft.Azure.Mobile.Ingestion.Models;
+using Microsoft.Azure.Mobile.Utils;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.Mobile.Test.Windows
+{
+    [TestClass]
+    public class StartServiceLogTests
+    {
+        private const string StorageTestChannelName = "storageTestChannelName";
+
+        [TestInitialize]
+        public void asdf()
+        {
+            LogSerializer.AddFactory(StartServiceLog.JsonIdentifier, new LogFactory<StartServiceLog>());
+        }
+
+        [TestMethod]
```

Comment:
Add comments to the [TestMethod]s that explain what they do (briefly, see other test files for example of this)